lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -1270,6 +1272,118 @@ def get_workflow_menu(
 class FastAPIInvocations:
     invocations_service: InvocationsService = depends(InvocationsService)
 
+    @router.post(
+        "/api/invocations/from_store",
+        name="create_invocations_from_store",
+        description="Create invocation(s) from a supplied model store.",
+    )
+    def create_invocations_from_store(
+        self,
+        payload: Annotated[CreateInvocationsFromStorePayload, CreateInvocationsFromStoreBody],
```

Comment:
This doesn't look right, you're annotating CreateInvocationsFromStorePayload with CreateInvocationsFromStoreBody, I don't know what that should mean ? Did you mean
`payload: CreateInvocationsFromStoreBody` ?